subscription-engine/kafka-ws/bin/entryPoint.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+# -e Exit imediatly if a comand exit with non-zero status
+# -x Print commands and their arguments as they are executed
+
+# Debug mode
+if [ ! -z "${DEBUG+x}" ]; then
+    set -x
+fi
+
+export KAKKA_WS_CONNECTION_RETRY_COUNT=${KAFKA_WS_APP_CONNECTION_RETRY_COUNT:-"5"}
+export KAFKA_WS_CONNECTION_RETRY_TIMEOUT=${KAFKA_WS_CONNECTION_RETRY_TIMEOUT:-"3"}
+
+export KAFKA_WS_CONSUMER_METADATA_BROKER_LIST=${KAFKA_WS_CONSUMER_METADATA_BROKER_LIST:-"kafka:9092"}
+
+# Redis parameters
+readonly REDIS_CONN_TIMEOUT=${REDIS_CONN_TIMEOUT:-"180"}
+readonly REDIS_HOST=${REDIS_HOST:-"redis"}
+readonly REDIS_PORT=${REDIS_PORT:-"6379"}
+readonly REDIS_PASSWD=${REDIS_PASSWD:-""}
```

Comment:
Shouldn't these variables follow the same pattern as those in branch `development`?

This is the documentation for branch `development`:

> **Redis**
> Redis database management configuration
> 
> | Key | Purpose   | Default Value   | Valid Values | Environment variable
> ------| ----------| ----------------|--------------|------------------------
> redis.host| Redis host | kafka-ws-redis | string | KAFKA_WS_REDIS_HOST
> redis.port | Redis port | 6379 | integer | KAFKA_WS_REDIS_PORT
> redis.db | Redis database | 1 | integer | KAFKA_WS_REDIS_DB

Reply:
By now, we shouldn't change this, the variable names will be changed in the next release.